diff/parse.go

```diff
@@ -59,23 +67,27 @@ func (r *MultiFileDiffReader) ReadFile() (*FileDiff, error) {
 		switch e := err.(type) {
 		case *ParseError:
 			if e.Err == ErrNoFileHeader || e.Err == ErrExtendedHeadersEOF {
-				return nil, io.EOF
+				trailing := ""
+				if fd != nil {
+					trailing = strings.Join(fd.Extended, "\n")
+				}
+				return nil, trailing, io.EOF
```

Comment:
Can you add a short commetn here that explains what's happening? You know, like "If we've reached the end and there's no diff, we've maybe parsed some content already, which is stored in fd.Extended, so we return that"

Reply:
I've not used that exact phrasing, but feel free to suggest changes to what I have pushed. 🙂

Reply:
That's great! Thanks!